core/block_svg.ts

```diff
@@ -795,7 +795,7 @@ export class BlockSvg
    * E.g. does trigger UI effects, remove nodes, etc.
    */
   override disposeInternal() {
-    if (this.isDeadOrDying()) return;
+    this.disposing = true;
```

Comment:
I am pretty sure that this one is redundant and can be removed, since you do a super call immediately and `Block.prototype.disposeInternal` also sets `this.disposing = true`.

Reply:
I agree, but to future-proof this (in case we ever add things before calling super) and make it extremely clear, I think it's reasonable to leave it in.

If this were more expensive I would remove it. But since it's cheap I think it's reasonable to leave.